ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -218,7 +222,7 @@ S3GetDirectory(const std::string &home, const std::string &name, bool create, st
 
     ret = stat(dirName.c_str(), &sb);
     if (ret != 0 && errno == ENOENT && create) {
-        (void)mkdir(dirName.c_str(), 0777);
+        UNUSED(mkdir(dirName.c_str(), 0777));
```

Comment:
I think we can just do `mkdir(dirName.c_str(), 0777)`. Thats my bad - I don't know why it was wrapped with void in the first place..

Reply:
Yep no problem! Will change this!